tests/unit-tests/test_nn_model.py

```diff
@@ -106,7 +139,26 @@ def test_rgat_with_zero_input(input_dim, output_dim):
     assert out["n3"].shape[1] == output_dim
     assert "n4" not in out
 
+@pytest.mark.parametrize("input_dim", [32])
+@pytest.mark.parametrize("output_dim", [32,64])
+def test_hgt_with_no_indegree_dstnodes(input_dim, output_dim):
+    block, inputs, ntypes, etypes = create_dummy_no_indegree_test_graph(input_dim)
+
+    layer = HGTLayer(input_dim,
+                     output_dim,
+                     ntypes,
+                     etypes,
+                     num_heads=4)
+    outputs = layer(block, inputs)
+    
+    assert outputs['n0'].shape[0] == 256
+    assert outputs['n0'].shape[1] == output_dim
+    assert outputs['n1'].shape[0] == 64
+    assert outputs['n1'].shape[1] == output_dim
+    assert outputs['n2'].shape[0] == 64
+    assert outputs['n2'].shape[1] == output_dim
 
 if __name__ == '__main__':
     test_rgcn_with_zero_input(32,32)
-    test_rgat_with_zero_input(32,64)
\ No newline at end of file
+    test_rgat_with_zero_input(32,64)
+    test_hgt_with_no_indegree_dstnodes(32, 64)
```

Comment:
Can you also check whether hgt can pass the test_rgat_with_zero_input test?

Reply:
Added, but the outputs are different from RGCN and RGAT. Original HGT algorithm does not handle such case. We need to decide if we want to support 0 number of dst node.

Reply:
Confirm with Minjie, the expected outputs of zero edges are fixed, which is same as the rgcn and rgat.